extensions/git/src/repository.ts

```diff
@@ -80,15 +80,15 @@ export class Resource implements SourceControlResourceState {
 	}
 
 	get rightUri(): Uri {
-		return this.resources[1];
+		return this.resources[1] as Uri;
```

Comment:
I don't think this is a good idea -- this should either return `Uri | undefined` or it should throw.

Reply:
I checked and adding `undefined` doesn't (visibly) cause any issues.